test/verify/check-system-services

```diff
@@ -1396,6 +1396,65 @@ WantedBy=multi-user.target
         m.execute("systemctl stop test.service")
         self.check_service_details(["Read-only", "Disabled"], [], enabled=False, onoff=False, kebab=False)
 
+    @testlib.skipImage("no quadlet (pod) support", "debian-stable", "ubuntu-2204", "ubuntu-2404", "rhel-8-10")
+    @testlib.nondestructive
+    def testQuadlets(self):
+        m = self.machine
+        b = self.browser
+
+        busybox_image = m.execute("podman images --format '{{.Repository}}' | grep busybox").strip()
+        container_service = f"""
```

Comment:
"Description: zigbee2mqtt", "ContainerName=message-broker" and "PodName=has" could use a bag of consistency powder and become a bit more legible.